ministryofjustice/analytics-platform-ops
# Adds IAM role support to Rstudio

## What

Allows RStudio to use IAM Role-based credentials rather than explicit credential pairs.

## How to review

1. Check out this PR
2. Install or upgrade your RStudio instance to specify an IAM Role:
```
helm upgrade kerin-rstudio charts/rstudio -f chart-env-config/dev/rstudio.yml --set Username=$USERNAME --set AWS.IAMRole=users/$ENV_$USERNAME --set RStudio.Image.Tag=v1.2.3 --namespace user-$USERNAME --install --wait
```
3. Start a new R session
4. Verify that the modified aws.signature/aws.s3 libraries use the specified role
